test/600_proxy_docker_py_test.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Run docker-py test suite against the proxy"
+
+assert_raises "docker_on $HOST1 ps | grep weaveproxy" 1
```

Comment:
`start_suite` removes all containers from the test hosts, so this check is redundant